src/applications/facility-locator/tests/e2e/search.cypress.spec.js

```diff
@@ -31,7 +31,7 @@ Cypress.Commands.add('verifyOptions', () => {
 describe('Facility search', () => {
   before(function() {
     // This test crashes in Jenkins about 5% of the time.
-    if (!Cypress.env('CIRCLECI')) this.skip();
+    // /if (!Cypress.env('CIRCLECI')) this.skip();
```

Comment:
Please discus this change with the testing tools team, esp. @ddzz

Reply:
I am not planning to just enable it, however, I want find out if this is correct `crashing on Jenkins about one out of every 15-20 runs.` first

Reply:
@brangi I was able to recreate this failure locally and on Jenkins by running the test in a loop. There's an example in the [Cypress flaky test management document](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/testing/end-to-end/cypress-flaky-test-management.md#how-to-detect-flaky-tests).

Reply:
@ddzz , oh thanks for this, let me try it out

Reply:
@ddzz what do you mean by https://github.com/department-of-veterans-affairs/va.gov-team/issues/20178#issuecomment-781679897 `reduce map animations ` ? you mean for example when the map zooms in slowly (navigating to the location)

Reply:
AFAIK we are already Mocking Mapbox API calls https://github.com/department-of-veterans-affairs/vets-website/blob/947e2bb66414c0376ae9213bf6350e791e751ba8/src/applications/facility-locator/tests/e2e/search.cypress.spec.js#L49 - are we not?